multipart aof backup details without script (#1794)
redis/redis-doc: topics/persistence.md

If you run a Redis instance with only AOF persistence enabled, you can still perform backups.
Since Redis 7.0.0, AOF files are split into multiple files which reside in a single directory determined by the `appendddirname` configuration.

> During normal operation all you need to do is copy/tar the files in this directory to achieve a backup, but if this is done during a [rewrite](#log-rewriting) you might end up with an ivalid backup.

Reviewer: ```suggestion
During normal operation all you need to do is copy/tar the files in this directory to achieve a backup. However, if this is done during a [rewrite](#log-rewriting), you might end up with an invalid backup.
```

To work around this you must disable AOF rewrites during the backup:

> 1. Turn of automatic rewrites with<br/>

Reviewer: ```suggestion
1. Turn off automatic rewrites with<br/>
```

   `CONFIG SET` `auto-aof-rewrite-percentage 0`<br/>
   Make sure you don't manually start a rewrite (using `BGREWRITEAOF`) during this time.
2. Check there's no current rewrite in progress using<br/>
   `INFO` `persistence`<br/>
   and verifying `aof_rewrite_in_progress` is 0. If it's 1, then you'll need to wait for the rewrite to complete.
3. Now you can safely copy the files in the `appenddirname` directory.
4. Re-enable rewrites when done:<br/>
   `CONFIG SET` `auto-aof-rewrite-percentage <prev-value>`

> **Note:** If you want to minimize the time AOF rewrites are disabled you may create hard links to the files in `appenddirname` (in step 3 above) and then re-enable rewites (step 4) after the hard links are created.
> Now you can copy/tar the hardlinks and delete them when done.

Reviewer: ```suggestion
**Note:** If you want to minimize the time AOF rewrites are disabled you may create hard links to the files in `appenddirname` (in step 3 above) and then re-enable rewites (step 4) after the hard links are created.
Now you can copy/tar the hardlinks and delete them when done. This works because Redis guarantees that it
only appends to files in this directory, or completely replaces them if necessary, so the content should be
consistent at any given point in time.
```
